src/animation.js

```diff
@@ -323,13 +322,13 @@ class AnimationPlayer {
 
     // Complete or start next cycle.
     if (normLinearTime == 1) {
-      this.complete_(/* success */ true, /* dir */ 0);
+      this.complete_(null, /* dir */ 0);
     } else {
       if (this.vsync_.canAnimate(this.contextNode_)) {
         this.task_(this.state_);
       } else {
         dev().warn(TAG_, 'cancel animation');
-        this.complete_(/* success */ false, /* dir */ 0);
+        this.complete_(new Error('cancel animation'), /* dir */ 0);
```

Comment:
I believe the intent is to simply give more info to our loggers and error reporting that this is not a "real" error, but rather an "abort" condition.